pypeit/core/framematch.py

```diff
@@ -38,6 +38,7 @@ def __init__(self):
                      ('pinhole', 'Pinhole observation used for tracing slit centers'),
                    ('pixelflat', 'Flat-field exposure used for pixel-to-pixel response'),
                    ('illumflat', 'Flat-field exposure used for illumination flat'),
+                   ('thermalflat', 'Flat-field exposure with lamps off used to remove thermal emission from dome'),
```

Comment:
Let's discuss the name. As I've said, thermal emission from the dome is only done in the k-band. In the Y through H one uses these flats to remove persistence. It would be more clear if your naming and if your descriptions were more general than the specific K-band usage case. 

I suggest calling them lampoffflats, and here write "to remove persistence from lamp on flat exposures and/or  thermal emission from the telescope and dome."

Reply:
changed to `lampoffflats`